"""A custom exception for showing error message"""
class CustomError(Exception):
    pass